src/auth/interceptors/createRetryInterceptor.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+const createRetryInterceptor = (options = {}) => {
+  const {
+    httpClient = axios.create(),
+    // This default algorithm is a recreation of what is documented here
+    // https://cloud.google.com/storage/docs/exponential-backoff
+    getBackoffMs = (nthRetry, maximumBackoff = 32000) => {
+      const randomSecond = Math.random();
+      const exponentialBackoff = ((2 ** nthRetry) + randomSecond) * 1000;
```

Comment:
I think it's okay as is, but it is mixed units (seconds and milliseconds). I was avoiding multiplying multiple things by 1000 but it may be best to make things clearer here. I'll refactor/reword and maybe you can take a second look?

Reply:
Agh. My brain wouldn’t compute it yesterday, but it sunk in today. Not much renaming is needed, but it would be nice.

Reply:
fixed